build.gradle

```diff
@@ -55,4 +55,30 @@ dockerCompose {
         projectName = null
         useComposeFiles = ["docker-compose-postgres.yml"]
     }
+
+    javaDevelopment {
+        projectName = null
+        useComposeFiles = ["docker-compose-java-development-build.yml"]
+    }
+
+    javaDevelopmentImage {
+        projectName = null
+        useComposeFiles = ["java-development/docker-compose-java-development-build.yml"]
+    }
 }
+
+task prepareJavaDevelopmentDockerImage(type:Exec) {
+    commandLine './prepare-java-development-docker-image.sh'
+}
+
+javaDevelopmentImageComposeBuild.dependsOn(prepareJavaDevelopmentDockerImage)
+
+task buildAndStartServicesMySql(type: GradleBuild) {
```

Comment:
This name is not consistent with `eventuate-tram-examples-customers-and-orders`, which uses `buildAndStartServicesMySqlBinlog` - I want to have consistent instructions for all projects. `Binlog` suffix can be removed.

Reply:
ok, I will remove it from eventuate-tram-examples-customers-and-orders